src/Lykke.Job.BlockchainCashoutProcessor/Modules/CqrsModule.cs

```diff
@@ -16,6 +17,8 @@
 using Lykke.Messaging.Contract;
 using Lykke.Messaging.RabbitMq;
 using Lykke.Messaging.Serialization;
+using CashinCompletedEvent = Lykke.Job.BlockchainCashoutProcessor.Wrokflow.Events.CashinCompletedEvent;
```

Comment:
suggest to use BlockchainCashoutProcessor.Wrokflow.Events implicity - this makes easier to understand that there are usage bounded context of another service